bazel/go_repositories.bzl

```diff
@@ -2151,8 +2151,8 @@ def go_repositories():
     go_repository(
         name = "org_golang_google_grpc",
         importpath = "google.golang.org/grpc",
-        sum = "h1:o1bcQ6imQMIOpdrO3SWf2z5RV72WbDwdXuK0MDlc8As=",
-        version = "v1.36.0",
+        sum = "h1:zWTV+LMdc3kaiJMSTOFz2UgSBgx8RNQoTGiZu3fR9S0=",
```

Comment:
this seems a spurious change? why do we need to change grpc version to increase the timeout to two hours?

Reply:
arg, let me create a new branch after rebasing from master to clean this up